frontend/packages/dev-console/src/components/add/AddPageLayout.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../../../../public/style/vars';
+
```

Comment:
Hmm, but we should try to avoid this cross package dependency when possible. How does it looks with 14 px? Maybe we / UX is fine with this as well? Can you prepare a screenshot with `var(--pf-global--FontSize--sm)` and share this with the UX team in [#tmp-odc-add](https://coreos.slack.com/archives/CHC2R6AGG)? Thanks :)

Reply:
--pf-global--FontSize--sm is suppose to add 14px as per the document but it is adding 13px.. for 13px text looks like below. You can compare the below one and previous screenshot I attached. In order to keep it to 14px, I used font-size-base variable.  

<img width="1417" alt="image" src="https://user-images.githubusercontent.com/102503482/162711103-2ed764c1-1eff-4c05-aead-d877e9e92ff6.png">

Reply:
I feel 13px looks small besides switch. So I kept 14px and before also it was 14px. Default label font-size is 16px in switch component. If we try to keep it to 16px, then that will get highlight, that is not necessary I guess. I can check in the channel you suggested.

Reply:
Okay thanks for asking in the channel and you explanation there. Maybe then 14 is fine. I will checkout your PR for a local test.